### DISCUSSION TITLE
驼峰的文档命名是否有配置修改能不转化成-的约定式路由的形式，还是直接使用驼峰的路由

### DISCUSSION THREAD

**Discussion by Liu-0114** [umijs/dumi#1940] [Q&A] [answered]
为了使得 URL 更加友好，dumi 会自动将文件路径中的驼峰命令（CamelCase）转换为中划线命名（kebab-case），例如 docs/HelloWorld.md 最终会生成 /hello-world。

目标，路由还是会生成/HelloWorld

**Comment by PeachScript** [comment, marked as answer]
参考：https://d.umijs.org/config#forcekebabcaserouting

**Comment by Liu-0114** [comment]
dumi已经下载到最新2.2.13版本了，但是不支持这个字段的配置，项目都运行不起来，
<img width="1440" alt="image" src="https://github.com/umijs/dumi/assets/59878300/25115c88-adef-44ca-92cb-ed26f2f399ca">

**Reply by PeachScript** [reply]
它是 resolve 的子项
